Package.resolved

```diff
@@ -10,6 +10,15 @@
           "version": "4.1.0"
         }
       },
+      {
+        "package": "BrightFutures",
```

Comment:
Why are these diffs in this PR?

Reply:
+1, it seems these are unnecessary packages.

Reply:
Ouch, true. I'll revert them.

Reply:
You reverted it, but restored again.🤣